### ISSUE TITLE
search.Result.getAllValues() returns an incomplete type

### ISSUE THREAD

**Issue by cxa-croper** [headintheclouddev/typings-suitescript-2.0#177]
Result.getAllValues() is currently typed as 

```typescript
{ [fieldId: string]: string; }
```

when it practice it's more like

```typescript
{
  [fieldId: string]: string & {
    value: string,
    text: string
  }[]
}
```

Many fields that reference other records will return an array of objects, each of which contain the record's internal ID as the value property and text related to the record (such as the entity or item name) as the text property.

**Comment by MrRob** [comment]
@cxa-croper Thank you for reporting this, fixed in version 2021.1.3!

**Comment by cxa-croper** [comment]
@MrRob Great! And so swiftly :-) thank you
